### ISSUE TITLE
Warning: PropTypes has been moved to a separate package.

### ISSUE THREAD

**Issue by dhaspden** [sconxu/react-native-checkbox#65]
Hello!

I have started a new React Native project and noticed this warning when I started using this library.  It isn't a huge deal to me and 16 is still in alpha.  Something to consider in the near future though.  I can submit a pull request if you want.

Thanks for the great work!

Full warning:

```
Warning: PropTypes has been moved to a separate package. 
Accessing React.PropTypes is no longer supported and will be removed completely in React 16. 
Use the prop-types package on npm instead. (https://fb.me/migrating-from-react-proptypes)
```

**Comment by dannyharding10** [comment]
This has been fixed in #66
